skore/src/skore/project.py

```diff
@@ -87,9 +88,9 @@ def put_several(
 
         Parameters
         ----------
-        key_to_value : dict[str, Any]
+        key_to_value: dict[str, Any]
             The key-value pairs to put in the Project. Keys must be strings.
-        on_error : "warn" or "raise", optional
+        on_error: "warn" or "raise", optional
```

Comment:
Please keep the numpy docstring convetion https://numpydoc.readthedocs.io/en/latest/format.html#parameters .